mitodl/mitxonline
# feat: program certificates

#### Pre-Flight checklist


#### What are the relevant tickets?
#1037 #1038 

#### What's this PR do?
Feature: Program Certificates

#### How should this be manually tested?
- Checkout to branch and apply migrations
- Create program and courses for that program
- Enroll in all courses
- Generate certificates for all courses 
- And now you'll be able to generate the certificate for the program in the Django admin
- Enable `Program Ui` and go to the dashboard
- You'll see the `View certificate` button, click and verify the certificate

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
<img width="1786" alt="Screenshot 2022-10-06 at 2 42 57 PM" src="https://user-images.githubusercontent.com/77275478/194316304-439c286e-472d-431c-bf99-51cd521dde41.png">
<img width="881" alt="Screenshot 2022-10-06 at 2 14 09 PM" src="https://user-images.githubusercontent.com/77275478/194316692-ff9cef6a-d7a3-4955-a41b-9778a157a3b1.png">
<img width="1164" alt="Screenshot 2022-10-06 at 6 06 03 PM" src="https://user-images.githubusercontent.com/77275478/194320479-0529bcde-9f7b-447f-963b-43b56797ec71.png">
<img width="255" alt="Screenshot 2022-10-06 at 2 34 10 PM" src="https://user-images.githubusercontent.com/77275478/194316733-739f18e9-984d-4851-8bdb-351088cab847.png">
<img width="287" alt="Screenshot 2022-10-06 at 2 34 52 PM" src="https://user-images.githubusercontent.com/77275478/194316737-e5fa5b88-7bb1-4e94-a0e3-e62f43e64728.png">
